### ISSUE TITLE
CoreData has no member perform

### ISSUE THREAD

**Issue by martheli** [JohnEstropia/CoreStore#165]
I just started using CoreStore and when trying to use the sample code, I am receiving this error. I also have import CoreStore at the top.  Below is the code:

```
CoreStore.perform(
            asynchronous: { (transaction) -> Void in
                let person = transaction.create(Into<Person>())
                person.name = "John Smith"
                person.age = 30
        },
            completion: { _ in }
        )

```

**Comment by JohnEstropia** [comment]
@martheli Hi, sorry about this. You are probably using the `master` branch which is still at CoreStore 3.1.1, while the README visible from github is using the `develop` branch which is at CoreStore 4.0.0.

If you wish to keep using the `master` branch (which is the latest on Cocoapods and Carthage), you can check the [README file here](https://github.com/JohnEstropia/CoreStore/tree/master).

If you wish to use CoreStore 4.0 (which is currently in final stages of beta), you can use the `develop` branch as is, or install using 
Cocoapods Podfile: 
```
pod 'CoreStore', :git => 'https://github.com/JohnEstropia/CoreStore.git', :branch => 'develop'
```
Carthage Cartfile:
```
github "JohnEstropia/CoreStore" "develop"
```
CoreStore 4.0 is ready but I'm still finishing up the documentation. Once I get it done I'll update for Cocoapods and Carthage as well.

**Comment by JohnEstropia** [comment]
@martheli Okay so I merged the CoreStore 4.0 update to the `master` branch as well. Let me know how it works for you.
